src/_gettsim/parameters/arbeitsl_geld_2.yaml

```
@@ -307,14 +307,14 @@ anteil_regelsatz:
   2005-01-01:
     zwei_erwachsene: 0.9
     weitere_erwachsene: 0.8
-    kinder_0_6: 0.6
-    kinder_7_13: 0.6
+    kinder_0_5: 0.6
```

```suggestion
    kinder_bis_5: 0.6
```
Any reason we do not use the naming convention we have elsewhere? If easy to change, please do, else disregard.